aws/resource_aws_cloudwatch_log_metric_filter.go

```diff
@@ -107,7 +113,7 @@ func resourceAwsCloudWatchLogMetricFilterUpdate(d *schema.ResourceData, meta int
 		return fmt.Errorf("Creating/Updating CloudWatch Log Metric Filter failed: %s", err)
 	}
 
-	d.SetId(d.Get("name").(string))
+	d.SetId(logGroupName + ":" + name)
```

Comment:
Please note that we consider changing the resource ID a breaking change, since it affects any configuration references and can show differing values depending on the provider version.

```suggestion
	d.SetId(d.Get("name").(string))
```

Reply:
Oh, I see. Thank you for the kind explanation.